planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -668,6 +668,7 @@ void ObstacleStopPlannerNode::searchObstacle(
   const VehicleInfo & vehicle_info, const StopParam & stop_param,
   const sensor_msgs::msg::PointCloud2::SharedPtr obstacle_ros_pointcloud_ptr)
 {
+  if (!object_ptr_) return;
```

Comment:
I'll fix this point in https://github.com/autowarefoundation/autoware.universe/pull/1330 in order to prevent conflict.

Reply:
And, I think that products which don't support perception will not stop in front of the obstacles.

Reply:
The perception is necessary in behavior velocity planner too and products which don't support perception uses empty object publisher so this is not a problem

Reply:
ref: https://github.com/autowarefoundation/autoware.universe/blob/ff2d01c7656ae6d54df8fc9fafc058cb9b21f2b6/planning/behavior_velocity_planner/src/node.cpp#L222

Reply:
Oh, I see :eyes: Thanks :+1: